aikau/src/main/resources/alfresco/core/ResizeMixin.js

```diff
@@ -61,7 +61,9 @@ define(["dojo/_base/declare",
          // Fire a custom event to let contained objects know that the node has been resized.
          this.throttledPublish(this.alfResizeNodeTopic, {
             node: resizedNode
-         }, true);
+         }, true, false, {
+            timeoutMs: 100
```

Comment:
Perhaps, but who are we to decide this. If we abstract to a configurable variable then at least it gives the option. Maybe some people do want resize events to occur more often that 100ms. Maybe there will be cases where some widgets want to check more frequently than than the default and others that don't.